### DISCUSSION TITLE
Sharing the URL

### DISCUSSION THREAD

**Discussion by LilaDarkstar** [beakerbrowser/beaker#1843] [Q&A]
Hi all, sorry if this is a stupid question... I've got the point of making a webpage and wanting to share it, the URL starts with hyper://.... and is super long so I was hoping to shorten it first. Through tinyurl, I made a much shorter one but when I paste it into a browser, it doesn't re-direct to the page. 

This is probably ridiculously obvious to those who know about these kinds of things, but will my page made in Beaker not be visible on say, Google or Internet Explorer, and only in Beaker? Can I make it visible in another browser somehow or is the only way that people can access it by downloading Beaker and going through that browser?

Thanks in advance!

**Comment by grondilu** [comment]
I'm pretty sure tinyurl is not designed to handle hyper urls.  You just can't significantly shorten an hyper:// url as it's a public key if I understand correctly.  Its entropy is too high.

As for visibility outside of Beaker, again as I understand correctly it is indeed not visible unless you use a gateway of some sort.

**Comment by pfrazee** [comment]
Hi @LilaDarkstar, you and @grondilu are correct, `hyper://` URLs require a browser that explicitly supports Hypercore. It's definitely one of the weaknesses of trying to do a new web protocol!
